Client/Frontend/Browser/BrowserLocationView.swift

```diff
@@ -128,16 +133,20 @@ class BrowserLocationView : UIView, UIGestureRecognizerDelegate {
         didSet {
             lockImageView.hidden = (url?.scheme != "https")
             let t = url?.absoluteString
+
             if t?.hasPrefix("http://") ?? false {
                 locationLabel.text = t!.substringFromIndex(advance(t!.startIndex, 7))
             } else if t?.hasPrefix("https://") ?? false {
                 locationLabel.text = t!.substringFromIndex(advance(t!.startIndex, 8))
             } else if t == "about:home" {
-                let placeholderText = NSLocalizedString("Search or enter address", comment: "The text shown in the URL bar on about:home")
-                locationLabel.attributedText = NSAttributedString(string: placeholderText, attributes: [NSForegroundColorAttributeName: UIColor.grayColor()])
+                // This will show the placeholder label instead of the url.
+                url = nil
+                locationLabel.attributedText = BrowserLocationView.PlaceholderText
+                return
```

Comment:
Setting the url = nil above will call through here and set it already. I was just trying to avoid some churn.